galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/events/IEvent.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.spi.events;
+
+public interface IEvent {
+
+    String getId();
+
+    void setId(String id);
+
+    String getTimestamp();
+
+    void setTimestamp(String timestamp);
+
+    String getMessage();
+
+    void setMessage(String message);
+
+    @Override
+    String toString();
```

Comment:
Minor change - we won't need to add `toString` here since it's defined in every object.

Reply:
Hi Eamonn, thank you for spotting this. I have overridden this method in the *Event classes that implement this interface so they output a formatted string describing the event but you're right it's not needed in this interface. I'll remove it.